tests/test_fauxfloat.py

```diff
@@ -45,13 +45,32 @@ def test_unary_ops(operation, num):
     num=st.floats(allow_nan=False, allow_infinity=False),
     other=st.floats(allow_nan=False, allow_infinity=False),
 )
-def test_binary_ops(operation, num, other):
+def test_binary_ops_float(operation, num, other):
     t = get_thingy(num)
 
     assert operation(t, other) == operation(num, other)
     assert operation(other, t) == operation(other, num)
 
 
+@pytest.mark.parametrize(
+    "operation",
+    [
+        operator.lt, operator.le, operator.eq, operator.ne, operator.ge,
+        operator.gt, operator.add, operator.mul, operator.sub,
+    ],
+)
+@given(
+    num=st.floats(allow_nan=False, allow_infinity=False),
+    other=st.floats(allow_nan=False, allow_infinity=False),
+)
+def test_binary_ops_thingy(operation, num, other):
```

Comment:
I sent a better proposal as #281.

Reply:
PS: This is missing a `test_binary_ops_thingy_nonzero`, but please don't actually do this  >_>'

Reply:
PS: This is missing a `test_binary_ops_thingy_nonzero`, but please don't actually do this  >_>'